samples/swift/uidemo/Samples/FIRStorageViewController.swift

```diff
@@ -45,15 +46,15 @@ class FIRStorageViewController: UIViewController {
     self.imageView.image = nil
     guard let text = self.textField.text else { return }
     guard let url = NSURL(string: text) else { return }
-    guard let components = url.pathComponents else { return }
 
-    self.storageRef = FIRStorageReference()
+    self.storageRef = FIRStorage.storage().referenceWithPath(url.absoluteString)
```

Comment:
I don't think this will work, since the url absoluteString will include everything instead of just the path, you want `url.path`